core/src/main/kotlin/com/exactpro/th2/read/db/core/util/StatementUtils.kt

```diff
@@ -65,7 +64,10 @@ private fun PreparedStatement.setManual(paramIndex: Int, value: String?, type: S
     }
 }
 
-private fun String.toTimestamp(): Timestamp = Timestamp.valueOf(this)
+private fun String.toTimestamp(): Timestamp {
+    return runCatchingException { Timestamp.from(Instant.parse(this)) }
+        .getOrElse { Timestamp.valueOf(this) }
```

Comment:
suggestion: maybe instead of catching the exception we could try the following - check for `T` in the string and choose the right method depending on that